tests/dashboard_api_tests.py

```diff
@@ -423,3 +423,40 @@ def test_get_related_fail(self):
 
         rv = self.client.get(uri)
         self.assertEqual(rv.status_code, 404)
+
+    def test_export(self):
+        """
+            Dashboard API: Test dashboard export
+        """
+        self.login(username="admin")
+        argument = [1, 2]
+        uri = f"api/v1/dashboard/export/?q={prison.dumps(argument)}"
```

Comment:
So the final URL here looks like `api/v1/dashboard/export/?q=!(1,2)`?

Reply:
exactly